out_writeapi.go

```diff
@@ -276,14 +276,12 @@ func FLBPluginFlush(data unsafe.Pointer, length C.int, tag *C.char) int {
 func FLBPluginFlushCtx(ctx, data unsafe.Pointer, length C.int, tag *C.char) int {
 	// Get Fluentbit Context
 	id := output.FLBPluginGetContext(ctx).(int)
-	log.Printf("[multiinstance] Flush called for id: %d", id)
-
 	// Locate stream in map
 	// Look up through reference
 	config, ok := configMap[id]
 	if !ok {
-		log.Printf("Skipping flush because config is not found for tag: %d.", id)
-		return output.FLB_OK
+		log.Fatal("Error in finding configuration")
```

Comment:
Also include the id in the log message. Also mention the issue occurred under Flush.

Reply:
This was fixed.